Applications/ProductionManager/Contracts/Import/Order.cs

```diff
@@ -0,0 +1,48 @@
+﻿
+using Newtonsoft.Json;
+
+namespace HomagConnect.ProductionManager.Contracts.Import
+{
+    /// <summary>
+    /// Order data
+    /// </summary>
+    public class Order
+    {
+        /// <summary>
+        /// The unique id of this order
+        /// </summary>
+        [JsonProperty(Order = 1)]
+        public string Id { get; set; } = string.Empty;
+
+        /// <summary>
+        ///  The name of this order
+        /// </summary>
+        [JsonProperty(Order = 2)]
+        public string Name { get; set; } = string.Empty;
+
+        /// <summary>
+        /// The name of the customer of this order
+        /// </summary>
+        [JsonProperty(Order = 3)]
+        public string CustomerName { get; set; } = string.Empty;
+        /// <summary>
+        /// The date the order was created at
+        /// </summary>
+
+        [JsonProperty(Order = 4)]
+        public DateTimeOffset OrderDate { get; set; }
+
+        /// <summary>
+        /// The delivery date of this order
+        /// </summary>
+
+        [JsonProperty(Order = 5)]
+        public DateTimeOffset? DeliveryDate { get; set; }
+
+        /// <summary>
+        /// The state of the order. Possible states are:`New` / `ReadyForProduction` / `InProduction` / `Finished` 
+        /// </summary>       
+        [JsonProperty(Order = 4)]
+        public string State { get; set; } = string.Empty;
```

Comment:
Please replace with an enumeration.

Reply:
you mean something like this? 
![image](https://github.com/HomagGroup/HOMAG-Connect/assets/47099441/91d1ba20-66a5-4dcc-bd06-b95b4e4a79e9)

Reply:
No, isn't there a enum "OrderState" or something like that? I just do not want to return a string in this case.

Reply:
Updated